app/serializers/project_serializer.rb

```diff
@@ -54,16 +56,47 @@ def self.paging_scope(params, scope, context)
     if context[:cards]
       scope.preload(:avatar)
     else
-      # this is an experiement to see if the workflow association
-      # preload is causing high CPU
-      # if this lingers past 2nd April 2020 - remove it and the flipper key
-      if Panoptes.flipper[:test_preload_workflow_associations].enabled?
-        scope = scope.preload(:workflows, :active_workflows)
-      end
       super(params, scope, context)
     end
   end
 
+  # override the serialize page method to preload the
+  # workflow and active_worklfows association data
+  # however avoid loading large json attributes
+  # instead load the data required for serializing
+  # the project and links
+  def self.serialize_page(page, options)
+    project_ids = page.pluck(:id)
```

Comment:
I think i should wrap this with the existing flipper test in case it causes problems and we want to change the behaviour at runtime. 
`if Panoptes.flipper[:test_preload_workflow_associations].enabled?`